README.md

```diff
@@ -40,9 +40,15 @@ Write a Chomsky type 2 grammar that describes any valid path containing at
 least one paver. Use `S` as the start state, and `r`, `g`, `b` as terminals that
 represent the tiles.
 
+
+
 ## A1.1
 
-«replace this with your answer»
+S -> T
+S -> T S T
+T -> 'r'
+T -> 'g'
+T -> 'b'
```

Comment:
z# 3

But there's no guarantee that both Ts in "T S T" will expand to the same color. Instead you need something like:

       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b